Source/SmallBasic.Editor/Libraries/Graphics/TurtleGraphicsObject.cs

```diff
@@ -22,6 +22,8 @@ public override void ComposeTree(TreeComposer composer)
                 attributes: new Dictionary<string, string>
                 {
                     { "href", $"Turtle.svg" },
```

Comment:
I suggest adding a comment above height/width to indicate that they're required on Firefox.